src/server/src/modules/facultyMember.ts

```diff
@@ -0,0 +1,42 @@
+import { IFacultyMember, FacultyMember } from '@entities/facultyMember';
+import { getRepository } from 'typeorm';
+import { promises } from 'fs-extra';
+/**
+ * @description saves a new faculty member profile in the database
+ * @param email string
+ * @param biography string
+ * @param firstName string
+ * @param middleName string
+ * @param lastName string
+ * @param departmentId string
+ * @param websiteLink string
+ * @param office string
+ * @param title string
+ * @returns Promise
+ */
+export const createFacultyMember = (
```

Comment:
I think this function should be a "updateFacultyMember", meaning it should update the information of an already existing faculty member. This because my 'sign-up' route will be the one responsible for actually creating a faculty member with some initial data(names, email, password). Then, when the faculty member has been registered, he/she can update his/her information by adding the things that you get in this function like biography, department, etc.

Reply:
Do you mean we want two APIs?
1. An API that takes in initial data (names, email, and password) and creates a professor profile
2. An API to update the professor profile with the detailed info

If so, do we want to add the "password" field to the facultyMember entity?

Reply:
Yes. the API that takes initial data and creates a professor record is handled by my authentication API (the authentication task that I'm working on). The API for updating a professor profile is what I think this task should be. I added the password fields to student and facultyMember entities in my user authentication task. I will be done with it by this afternoon and will assign you as a reviewer.

Reply:
How do we update "user" field inside "facutlyMember"?

getRepository(FacultyMember).update(id, {
            departmentId,
            websiteLink,
            office,
            title
        });
or
getRepository(FacultyMember).update(id, {
            departmentId,
            websiteLink,
            office,
            title,
            user
        });
makes "user" field of "facultyMember" null.

Reply:
Done.